packages/apps/public/locales/ru/translation.json

```
@@ -22,7 +22,7 @@
   "Account balance:": "Баланс аккаунта",
   "Accounts injected from any of these extensions will appear in this application and be available for use. The above list is updated as more extensions with external signing capability become available.": "Счета, введенные из этих расширений, могут исспользоваться в нашем приложении. Список дополняется по мере добавления новых расширений с внешними возможностями подписей.",
   "Active nominations ({{count}})": "Активные номинации ({{count}})",
-  "Add a tip to this extrinsic, paying the block author for greater priority": "Добавить чаевые для автора блока для большей приоретизации",
+  "Add a tip to this extrinsic, paying the block author for greater priority": "Добавыть чаевые для автора блока для большей приоритезации",
```

```suggestion
  "Add a tip to this extrinsic, paying the block author for greater priority": "Добавить чаевые для автора блока для большей приоритизации",
```
https://ru.wiktionary.org/wiki/%D0%BF%D1%80%D0%B8%D0%BE%D1%80%D0%B8%D1%82%D0%B8%D0%B7%D0%B0%D1%86%D0%B8%D1%8F